climada/hazard/low_flow.py

```diff
@@ -48,79 +47,67 @@
 HAZ_TYPE = 'LF'
 """Hazard type acronym for Low Flow / Water Scarcity"""
 
-FILENAME_NC = '%s_%s_%s_%s_%s_%s_%s.nc'  #
-# %(gh_model, cl_model, scenario, soc, fn_str_var, yearrange)
-
-GH_MODEL = ['H08',
-            'CLM45',
-            'ORCHIDEE',
-            'LPJmL',
-            'WaterGAP2',
-            'JULES-W1',
-            'MATSIRO'
-            ]
-CL_MODEL = ['gfdl-esm2m',
-            'hadgem2-es',
-            'ipsl-cm5a-lr',
-            'miroc5',
-            'gswp3',
-            'wfdei',
-            'princeton',
-            'watch',
-            ]
-
-SCENARIO = ['historical',
-            'rcp26',
-            'rcp60',
-            'hist']
-
-SOC = ['histsoc',
-       '2005soc',
-       'rcp26soc',
-       'rcp60soc',
-       'pressoc']
-
 FN_STR_VAR = 'co2_dis_global_daily'  # FileName STRing depending on VARiable
-# (according to ISIMIP filenaming)
+"""constant part of discharge output file (according to ISIMIP filenaming)"""
 
-
-# list of year chunks: multiple files are combined
 YEARCHUNKS = dict()
-# historical:
-YEARCHUNKS[SCENARIO[0]] = list()
+"""list of year chunks: multiple files are combined"""
+
+YEARCHUNKS['historical'] = list()
+"""historical year chunks ISIMIP 2b"""
 for i in np.arange(1860, 2000, 10):
-    YEARCHUNKS[SCENARIO[0]].append('%i_%i' % (i + 1, i + 10))
-YEARCHUNKS[SCENARIO[0]].append('2001_2005')
+    YEARCHUNKS['historical'].append(f'{i+1}_{i+10}')
+YEARCHUNKS['historical'].append('2001_2005')
 
-# rcp26:
-YEARCHUNKS[SCENARIO[1]] = ['2006_2010']
-for i in np.arange(2010, 2090, 10):
-    YEARCHUNKS[SCENARIO[1]].append('%i_%i' % (i + 1, i + 10))
-YEARCHUNKS[SCENARIO[1]].append('2091_2099')
-# rcp60:
-YEARCHUNKS[SCENARIO[2]] = YEARCHUNKS[SCENARIO[1]]
-# 'hist'
-YEARCHUNKS[SCENARIO[3]] = list()
+YEARCHUNKS['hist'] = list()
+"""historical year chunks ISIMIP 2a"""
 for i in np.arange(1970, 2010, 10):
-    YEARCHUNKS[SCENARIO[3]].append('%i_%i' % (i + 1, i + 10))
+    YEARCHUNKS['hist'].append(f'{i+1}_{i+10}')
+
+YEARCHUNKS['rcp26'] = ['2006_2010']
+for i in np.arange(2010, 2090, 10):
+    YEARCHUNKS['rcp26'].append(f'{i+1}_{i+10}')
+YEARCHUNKS['rcp26'].append('2091_2099')
+YEARCHUNKS['rcp60'] = YEARCHUNKS['rcp26']
+"""future year chunks"""
 
-REFERENCE_YEARRANGE = [1971, 2005]
+REFERENCE_YEARRANGE = (1971, 2005)
+"""default year range used to compute threshold (base line reference)"""
 
-TARGET_YEARRANGE = [2001, 2005]
+TARGET_YEARRANGE = (2001, 2005)
+"""default year range of hazard"""
```

Comment:
Which hazard?

Reply:
will change comment to:
`"""arbitrary default, i.e. default year range of historical low flow hazard 2001-2005"""`